packages/@lwc/features/src/flags.ts

```diff
@@ -18,6 +18,8 @@ const featureFlagLookup: FeatureFlagLookup = {
     ENABLE_NODE_LIST_PATCH: null,
     ENABLE_HTML_COLLECTIONS_PATCH: null,
     ENABLE_NODE_PATCH: null,
+    // Disables the fix for #2121
+    ENABLE_LEAKING_EVENTS: null,
```

Comment:
This name is pretty opaque. You can either change the flag name to make it more eloquent (I don't have any idea right now), or add a proper comment explaining what this is.

Reply:
what about "ENABLE_NON_COMPOSED_EVENT_LEAKAGE" ?

Reply:
Sounds better to me.